<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerAppClass</class>
 <widget class="QMainWindow" name="ServerAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>160</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="uidLineEdit">
       <property name="placeholderText">
        <string>uid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="temperatureLineEdit">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="placeholderText">
        <string>Temperatire</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LoginButton">
       <property name="text">
        <string>Set Temperature</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GetStatusButton">
       <property name="text">
        <string>Get Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ResetDefaultButton">
       <property name="text">
        <string>Reset Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ResetWIFIButton">
       <property name="text">
        <string>ResetWifi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RebootButton">
       <property name="text">
        <string>RebootButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DeviceNameLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeDeviceNameButton">
       <property name="text">
        <string>Change Device Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteDeviceName">
       <property name="text">
        <string>Delete Device Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PWLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeAdminPWButton">
       <property name="text">
        <string>Change Admin Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeUserPWButton">
       <property name="text">
        <string>Change User Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="serverapp.qrc"/>
 </resources>
 <connections/>
</ui>
